packages/open-next/src/adapters/routing/middleware.ts

```diff
@@ -123,18 +123,13 @@ export async function handleMiddleware(
   // If the middleware returned a Redirect, we set the `Location` header with
   // the redirected url and end the response.
   if (res.statusCode >= 300 && res.statusCode < 400) {
-    const location = result.response.headers
```

Comment:
iirc, we had to handle this case specifically for middleware `Redirects`. Can you verify that it works with this change?

Reply:
Yeah, i`ve tested it by patching 2.3.6 version. Maybe I need to cover this case by some tests? Do you have any contributing docs?

Reply:
I believe we have this covered (middleware redirect test)[https://github.com/sst/open-next/blob/main/packages/tests-e2e/tests/appRouter/middleware.redirect.test.ts]

App: https://github.com/sst/open-next/blob/main/examples/app-router/middleware.ts#L8